www/templates/page-template.js

```diff
@@ -26,12 +26,14 @@ class PageTemplate extends LitElement {
   }
 
   updated() {
+    console.log('ENTER page template updated', window.location.pathname);
     this.route = window.location.pathname;
```

Comment:
My understanding is this lifecycle is run after the component has been rendered. I tested using the connectedCallback() to set the this.page variable from the page-template and had better results (I wasn't able to break it). Maybe that's just chance, I don't know.

Reply:
`this.page` was actually undefined at that point, so `updated` seemed like a good way to make the sure the value is "ready".